app/js/lib/country_specifics_repository.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+var _ = require( 'underscore' ),
+	countrySpecifics = {
+		generic: {
```

Comment:
This could be outside countrySpecifics (and it is not actually country specific)